alternator/executor.cc

```diff
@@ -1571,7 +1583,8 @@ rmw_operation::rmw_operation(service::storage_proxy& proxy, rjson::value&& reque
 std::optional<mutation> rmw_operation::apply(foreign_ptr<lw_shared_ptr<query::result>> qr, const query::partition_slice& slice, api::timestamp_type ts) {
     if (qr->row_count()) {
         auto selection = cql3::selection::selection::wildcard(_schema);
-        auto previous_item = executor::describe_single_item(_schema, slice, *selection, *qr, {});
+        add_consumption_helper helper;
```

Comment:
I don't understand what I'm seeing here. Why pass a local variable if the variable is destroyed without being read?
If the point is that this variable just helps add consumption to some global counter - then why does it need to be a local variable? What's the point of constructing it on every call when it's just constructed the same?